internal/services/dataprotection/data_protection_backup_vault_resource.go

```diff
@@ -102,19 +100,34 @@ func resourceDataProtectionBackupVault() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice(backupvaults.PossibleValuesForSoftDeleteState(), false),
 			},
 
+			"immutability": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				Default:      false,
```

Comment:
This wants to be the following
```suggestion
				Default:      backupvaults.ImmutabilityStateDisabled,
```